### PR TITLE
*: update to use IANA-assigned ports

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

File: etcdctl/command/util.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "127.0.0.1:4001,127.0.0.1:2379"
```
Comment (yichengq) [inline etcdctl/command/util.go]: this is for backward compatibility. etcdctl should be able to do requests to old etcd cluster.

File: etcdctl/main.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ func main() {
 		cli.BoolFlag{Name: "debug", Usage: "output cURL commands which can be used to reproduce the request"},
 		cli.BoolFlag{Name: "no-sync", Usage: "don't synchronize cluster information before sending request"},
 		cli.StringFlag{Name: "output, o", Value: "simple", Usage: "output response in the given format (`simple` or `json`)"},
-		cli.StringFlag{Name: "peers, C", Value: "", Usage: "a comma-delimited list of machine addresses in the cluster (default: \"127.0.0.1:4001\")"},
```
Comment (yichengq) [inline etcdctl/main.go]: help to match this with the code?

File: pkg/netutil/netutil_test.go [hunk 3]
```diff
@@ -34,21 +34,21 @@ func TestResolveTCPAddrs(t *testing.T) {
 		{
 			urls: [][]url.URL{
 				[]url.URL{
-					url.URL{Scheme: "http", Host: "127.0.0.1:4001"},
+					url.URL{Scheme: "http", Host: "127.0.0.1:2379"},
 					url.URL{Scheme: "http", Host: "127.0.0.1:2379"},
```
Comment (yichengq) [inline pkg/netutil/netutil_test.go]: This is for testing on old and new ports. We should keep the old value.

File: etcdctl/command/util.go (hunk 1 above)
Comment (crawford) [inline etcdctl/command/util.go]: Woops. I missed that one.

File: etcdctl/main.go (hunk 2 above)
Comment (crawford) [inline etcdctl/main.go]: Sorry, I don't understand.

Comment (yichengq) [inline etcdctl/main.go]: (default: "127.0.0.1:4001,127.0.0.1:2379")

Comment (crawford) [inline etcdctl/main.go]: Fixed.

File: rafthttp/urlpick_test.go [hunk 4]
```diff
@@ -24,12 +24,12 @@ import (
...
 // and always returns the same one.
 func TestURLPickerPick(t *testing.T) {
-	picker := mustNewURLPicker(t, []string{"http://127.0.0.1:2380", "http://127.0.0.1:7001"})
+	picker := mustNewURLPicker(t, []string{"http://127.0.0.1:2380", "http://127.0.0.1:2380"})
 
 	u := picker.pick()
 	urlmap := map[url.URL]bool{
 		url.URL{Scheme: "http", Host: "127.0.0.1:2380"}: true,
-		url.URL{Scheme: "http", Host: "127.0.0.1:7001"}: true,
+		url.URL{Scheme: "http", Host: "127.0.0.1:2380"}: true,
```
Comment (yichengq) [inline rafthttp/urlpick_test.go]: we don't need to change this

Comment (crawford) [inline rafthttp/urlpick_test.go]: -_-  yes, you are correct. I thought I caught all of these.

**Comment by yichengq** [comment]
(●´∀｀●) lgtm
@crawford Thanks for the contribution!

**Pull request merged**